plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeConnectorSmokeTest.java

```diff
@@ -2161,6 +2167,55 @@ public void testPartitionFilterIncluded()
         }
     }
 
+    @Test(dataProviderClass = DataProviders.class, dataProvider = "trueFalse")
+    public void testConcurrentInsertsReconciliation(boolean partitioned)
```

Comment:
Can we run this test more than once as it's not deterministic?
As in for example: io.trino.plugin.iceberg.BaseIcebergConnectorSmokeTest#testDeleteRowsConcurrently

Reply:
One of the version had more runs. Any reason why it's only one run again?